packages/manager/modules/pci/src/components/hds/component.js

```diff
@@ -0,0 +1,15 @@
+import controller from './controller';
+import template from './template.html';
+
+const component = {
+  bindings: {
+    hds: '=',
+    model: '=',
+    onChange: '=',
+    onLinkClicked: '=',
```

Comment:
All two way bindings ? Are we sure it is needed ?

Reply:
No we don't need 2wb, it's a copy-past.
Thanks for.